test/vending/VendingMachine.test.js

```diff
@@ -305,90 +316,93 @@ describe("VendingMachine", () => {
 
       context("when updating part of what was previously wrapped", () => {
```

Comment:
*when unwrapping

Realized we have a typo here, I'll fix it in the original PR.